app-crypt/efitools/efitools-1.8.1.ebuild

```diff
@@ -12,22 +12,36 @@ SRC_URI="https://git.kernel.org/pub/scm/linux/kernel/git/jejb/efitools.git/snaps
 LICENSE="GPL-2 LGPL-2.1"
 SLOT="0"
 KEYWORDS="~amd64 ~arm64 ~x86"
-IUSE="libressl"
+IUSE="libressl static"
 
-RDEPEND="!libressl? ( dev-libs/openssl:0= )
-	libressl? ( dev-libs/libressl:0= )
+LIB_DEPEND="!libressl? ( dev-libs/openssl:0=[static-libs(+)] )
+	libressl? ( dev-libs/libressl:0=[static-libs(+)] )"
+
+RDEPEND="!static? ( ${LIB_DEPEND//\[static-libs(+)]} )
 	sys-apps/util-linux"
 
 DEPEND="${RDEPEND}
 	app-crypt/sbsigntool
 	dev-perl/File-Slurp
+	static? (
+		${LIB_DEPEND}
+		dev-util/pkgconfig
+	)
 	sys-apps/help2man
 	sys-boot/gnu-efi
 	virtual/pkgconfig"
 
-PATCHES=( "${FILESDIR}/${P}-libressl-compatibility.patch" )
+PATCHES=(
+	"${FILESDIR}/1.7.0-Make.rules.patch"
+	"${FILESDIR}/${P}-libressl-compatibility.patch"
+)
 
 src_prepare() {
+	if use static; then
+		append-ldflags -static
+		sed -i 's/-lcrypto/`pkg-config --static --libs libcrypto`/g' Makefile || die
```

Comment:
we never want to hardcode `pkg-config`.  you want to use `$(tc-getPKG_CONFIG)` from toolchain-funcs instead.